ta/crypto_perf/ta_crypto_perf.c

```diff
@@ -27,7 +27,7 @@ static uint8_t iv[] = { 0xA0, 0xA1, 0xA2, 0xA3, 0xA4, 0xA5, 0xA6, 0xA7,
 			0xA8, 0xA9, 0xAA, 0xAB, 0xAC, 0xAD, 0xAE, 0xAF };
 static int use_iv;
 
-static TEE_OperationHandle crypto_op;
+static TEE_OperationHandle crypto_op = TEE_HANDLE_NULL;
```

Comment:
This isn't needed, since it's in .bss it's cleared automatically.

Reply:
Yes. I have deleted it.